packages/target-browser/runtime-browser/runtime.ts

```diff
@@ -367,6 +367,21 @@ class BrowserRuntime implements Runtime {
       body: name,
     })
   }
+  async getJsonFromBase64(base64: string): Promise<any> {
+    try {
+      const text = atob(base64)
+      const length = text.length
+      const bytes = new Uint8Array(length)
+      for (let i = 0; i < length; i++) {
+        bytes[i] = text.charCodeAt(i)
+      }
+      const decoder = new TextDecoder()
+      return JSON.parse(decoder.decode(bytes))
+    } catch (error) {
+      this.log.critical('String could not de decoded or parsed')
+      return null
+    }
+  }
```

Comment:
How is this different from `JSON.parse(atob(base64))`?

Reply:
If you test it with an base64 encoded json

it converts the resulting output:

Works:
```
const a = 'eyJhIjogIsOcw5bDhD8gVGVzdCDwn5GNIn0='
function getJsonFromBase64(base64) {
    try {
      const text = atob(base64)
      const length = text.length
      const bytes = new Uint8Array(length)
      for (let i = 0; i < length; i++) {
        bytes[i] = text.charCodeAt(i)
      }
      const decoder = new TextDecoder()
      return JSON.parse(decoder.decode(bytes))
    } catch (error) {
      console.log('String could not de decoded or parsed')
      return null
    }
  }
getJsonFromBase64(a)

Output: {a: 'ÜÖÄ? Test 👍'}
```
```
const a = 'eyJhIjogIsOcw5bDhD8gVGVzdCDwn5GNIn0='
 JSON.parse(atob(a))
 
 Ouput: {a: 'Ã\x9CÃ\x96Ã\x84? Test ð\x9F\x91\x8D'}
 ```